RecoVertex/AdaptiveVertexFit/src/AdaptiveVertexFitter.cc

```diff
@@ -632,9 +638,14 @@ AdaptiveVertexFitter::fit( const vector<RefCountedVertexTrack> & tracks,
          ( ((previousPosition - newPosition).mag() > theMaxShift) ||
            (!(theAssProbComputer->isAnnealed()) ) ) ) ;
 
-  if ( theWeightThreshold > 0. &&  ns_trks < 2 && !withPrior ) 
+  if ( theWeightThreshold > 0. &&  ns_trks < 1 && !withPrior ) 
```

Comment:
Was causing a crash with the 4D vertex clustering. More likely it is reflective of a problem there.
Will double check.